src/main/java/controller/GetMessage.java

```diff
@@ -109,7 +114,14 @@ public void getMessageWithoutChatIdPathVariable(HttpServletRequest request) {
         }
 
         Message message = queryMessage.getMessage(messageId);
-    
-        return SuccessResponseGenerator.getSuccessResponseForGetMessage(message, userId);
+        Attachment attachment = null;
```

Comment:
You don't need to declare this variable here.
It can be within the if-block below.

Reply:
Done